### PR TITLE
Updated Commander Genius to v3.5.1. Refactored to current PM standards.

### PR DESCRIPTION
**Pull request by t0b10-r3tr0**
Updated Commander Genius to v3.5.1. Refactored to current PM standards.

# New Port for Commander Genius (v3.5.1)

## Game Information
- **Title**: Commander Genius
- **URL**: https://gitlab.com/Dringgstein/Commander-Genius

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [X] ArkOS
- [X] ROCKNIX
- [X] muOS

### Resolution Tests
Test all major resolutions:
- [X] 480x320
- [X] 640x480
- [X] 720x720 (RGB30)
- [X] Higher resolutions (e.g., 1280x720)

## File Structure
- The port contains the following structure:

  - commander.genius/
      - cgenius/
        - conf/
        - gamedata/
        - libs/
        - license/      
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Commander Genius.sh

## Additional Resources
[**Community created content**](https://keenwiki.shikadi.net/wiki/Main_Page) can be downloaded and added either manually or automatically (with internet connection) through the provided launcher.

### CODE REVIEW

File: ports/commander.genius/cgenius/CGeniusExe
Comment (cdeletre) [inline ports/commander.genius/cgenius/CGeniusExe:1]: Filesize has changed. Is it a new build ?

File: ports/commander.genius/cgenius/libs/.gitkeep
Comment (cdeletre) [inline ports/commander.genius/cgenius/libs/.gitkeep:1]: `libs` folder must be renamed to `libs.aarch64` accroding to the new launch script

**Review by cdeletre — CHANGES_REQUESTED** [review]
Good job ! few thing to adjust and I think we're good to go :)

File: ports/commander.genius/cgenius/CGeniusExe
Comment (t0b10-r3tr0) [reply ports/commander.genius/cgenius/CGeniusExe:1]: Yes. New minor version bump. Binary changed from version 3.5.0 to [3.5.1](https://gitlab.com/Dringgstein/Commander-Genius/-/releases/v3.5.1).

File: ports/commander.genius/cgenius/libs/.gitkeep
Comment (t0b10-r3tr0) [reply ports/commander.genius/cgenius/libs/.gitkeep:1]: There are no libs included with this port. Should I just remove the directory?

File: ports/commander.genius/screenshot.png
[the remark this answers is not in the document]
Comment (t0b10-r3tr0) [reply ports/commander.genius/screenshot.png:1]: Great tool. Also ran it on the cover.png.

File: ports/commander.genius/cgenius/libs/.gitkeep
Comment (t0b10-r3tr0) [reply ports/commander.genius/cgenius/libs/.gitkeep:1]: Renamed as suggested.

**Comment by t0b10-r3tr0** [comment]
I have resolved the identified issues and this is, once again, ready for review.

File: ports/commander.genius/cover.png
[the remark this answers is not in the document]
Comment (t0b10-r3tr0) [reply ports/commander.genius/cover.png:1]: This has been done. Also ran pngquant against it.

**Review by Cebion — CHANGES_REQUESTED** [review]
- See Discord for comments, (porter, description and instruction length)

**Comment by t0b10-r3tr0** [comment]
Please see updated `ports.json`. I've incorporated the suggestions as provided in the PM Discord. Thanks folks.

**Comment by t0b10-r3tr0** [comment]
I've completed the final tweaks to the documentation and ports.json and am confident this can be reviewed.

**Review by Cebion — APPROVED** [review]

**Review by cdeletre — APPROVED** [review]

**Pull request merged**